aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/ReadHandler.java

```diff
@@ -25,14 +26,17 @@
         }
 
         DescribeLogGroupsResponse response = null;
+        ListTagsLogGroupResponse tagsResponse = null;
         try {
             response = proxy.injectCredentialsAndInvokeV2(Translator.translateToReadRequest(model),
-                ClientBuilder.getClient()::describeLogGroups);
+                    ClientBuilder.getClient()::describeLogGroups);
+            tagsResponse = proxy.injectCredentialsAndInvokeV2(Translator.translateToListTagsLogGroupRequest(model.getLogGroupName()),
```

Comment:
To have backward compatibility, the ListTagsLogGroups call should be soft-fail if no permission for tagging.

Reply:
sure, I can add it, but I don't get why I should support here backward compatibility? Shouldn't I then do it everywhere, where I interact with tags in some way? I'm just really curious about it, hope to get some more insights 😄

Reply:
particularly an issue with `ReadHandler` since permissions are expanding without customer changes

Reply:
@gruebel  Some users may won't have tagging permissions, especially they use stack role and restrict the permissions to operate a stack . 

In this way, it will break people if this change go out.   It makes senses to fail if users specify tags during create or update time. However, if they creates the log group successfully before,  it will be bad experience to break them.  BTW, the READ will be used for `GetAtt` function to get the attribute data.

Reply:
Thanks for the insight. Makes sense with the restocked permission for stack updates. I also used it in my previous work.

Reply:
What I'm still curious about is, what is actually the List handler used for. The Read one I understand and it is used also during stack resource import, but the List one?!

Reply:
This List handler is not used in CloudFormation yet, it intends for future features, so having it hard fail for permission issue is right to do.